custom_components/hypervolt_charger/manifest.json

```diff
@@ -7,6 +7,6 @@
   "integration_type": "device",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/gndean/home-assistant-hypervolt-charger/issues",
-  "requirements": ["websockets==11.0.3", "aiofiles==24.1.0"],
+  "requirements": ["websockets<=14.0", "aiofiles==24.1.0"],
```

Comment:
Apologies but should still include the `>=11.0.3`. Separated by a comma should work but test it out on your instance to double check.

Reply:
oh right I got you, will test it now